### DISCUSSION TITLE
Azure Monitor data source, add proxy support

### DISCUSSION THREAD

**Discussion by BenoitPoulet** [grafana/grafana#40446] [Data Sources]
Hello,

as many servers are behind firewall, with access to filtered by HTTP proxy.  It would be nice to add HTTP proxy to the Azure data source : 
https://grafana.com/docs/grafana/latest/datasources/azuremonitor/

Regards.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
